pkg/util/mount/nsenter_mount.go

```diff
@@ -203,6 +204,33 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 	return true, nil
 }
 
+// Calling Open with O_EXCL without O_CREATE only supported on linux, so
+// we just return false here.
+func (n *NsenterMounter) DeviceOpened(pathname string) (bool, error) {
```

Comment:
`NsenterMounter` is still linux (`+build linux` at the top of the file). It is used for hyperkube deployments where k8s is run inside a container and therefore methods like mount are custom (ensuring mount occurs in host's root mount namespace). AFAIK this method should work the same as it does in the plain linux mounter.